preprocessing_pipeline/config.json

```diff
@@ -0,0 +1,30 @@
+{
+    "transform_nifti_to_npy": {
```

Comment:
Każda sekcja config'u - to konfiguracja dla etapu pipeline'u. Klucz - nazwa etapu, wartość niezbędne wartości